ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.html

```diff
@@ -185,7 +185,8 @@ <h3 class="Color--warn Line--Break">Lost to Follow-Up</h3>
             <td>Gender</td>
             <td>{{gender | titlecase}}</td>
           </tr>
-          <tr *ngIf="participant.data.profile['preferredLanguage'] != null && participant.data.profile['preferredLanguage'] !== ''">
+          <tr
```

Comment:
Unnecessary CR.

Reply:
Elliot, it was just formatted automatically, after using the keyboard shortcut - **options + command + L**.
 all is good here